neo/Network/P2P/Helper.cs

```diff
@@ -1,10 +1,43 @@
 ﻿using Neo.Network.P2P.Payloads;
 using System.IO;
+using System.IO.Compression;
 
 namespace Neo.Network.P2P
 {
     public static class Helper
     {
+        public static byte[] Uncompress(this byte[] data)
+        {
+            using (var output = new MemoryStream())
+            using (var input = new MemoryStream(data))
+            using (var gzip = new GZipStream(input, CompressionMode.Decompress))
+            {
+                int nRead;
+                byte[] buffer = new byte[1024];
+
+                while ((nRead = gzip.Read(buffer, 0, buffer.Length)) > 0)
+                {
+                    output.Write(buffer, 0, nRead);
+                }
+
+                return output.ToArray();
+            }
+        }
+
+        public static byte[] Compress(this byte[] data)
```

Comment:
@vncoelho I was referring with using zstandard it here instead.

Reply:
I think I did not understand yet.
Because I read something saying that this zstandard was optimized for small data, but its performance was enhanced with learning techniques by providing previous examples to the tool.

Reply:
We can do some testing and then choose the one that works best.

Reply:
Gzip is detectable by his header too, so maybe we can choose according to the magic numbers

Reply:
So `CompressedGzip` can be renamed to `Compressed`.

Reply:
@shargon, I think that it is better in the header instead of the magic, no?
In the header it would be easier to test other compression methods.